dashboard/test/helpers/application_helper_test.rb

```diff
@@ -1,98 +1,99 @@
 require 'test_helper'
 
 class ApplicationHelperTest < ActionView::TestCase
+  include ApplicationHelper
```

Comment:
did a bunch of tests (like "canonical_hostname in test") get deleted here? Was that intentional?